README.md

```diff
@@ -1,3 +1,11 @@
+# USBDeskbar
+
+![USBDeskbar](./images/screenshot.png)
+
+## Description
+
 A little tool to see and hear plug/unplug USB Device. and with popup menu you can see technical informations. gives info about USB device, for driver dev etc ...
```

Comment:
This is rather poor English with wrong capitalization to boot...

Reply:
Yep, because I wrote it, sure...
No, I only changed the format to .md and uploaded an image, because that's a GCI task.

Reply:
Yep, because I wrote it, sure...
No, I only changed the format to .md and uploaded an image, because that's a GCI task.

Reply:
Yep, because I wrote it, sure...
No, I only changed the format to .md and uploaded an image, because that's a GCI task.